src/github_utils.py

```diff
@@ -19,48 +19,56 @@ def get_pull_request_diff():
     # Get diff content
     diff_url = pull_request.diff_url
     response = requests.get(diff_url)
-    print(response.text)
```

Comment:
It is better to use logging here. Also, this line should be removed for production code.

```suggestion
Remove the commented print statement. Or replace it with logging.debug(response.text)
```